package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "build": "cpx index.d.ts dist/ && tsc -p .",
     "build:watch": "cpx index.d.ts dist/ && tsc -p . -w",
-    "clean": "rimraf dist/**/* && rimraf tests/simple/coverage && rimraf tests/simple-async/coverage",
+    "clean": "rimraf dist/**/* && rimraf tests/simple/coverage && rimraf tests/simple-async/coverage && rimraf tests/**/*/debug.txt",
     "clean-build": "npm run clean && npm run build",
```

Comment:
it might be better to create a separate directory for debug logs than to have them scattered all over and then delete them like this.

Reply:
Yeah I considered just adding the debug.txt in the project root, but we'd still need to delete them for each subproject yeah?

Reply:
Sorry about this. I didn't notice that the delete was recursive under the `tests` directory. This should be fine. I was somehow under the impression that this will be done throughout the project.

I now have a different question. These logs will be produced when debugging sample repos, right? If so, will the logs still be placed under ts-jest in node_modules?

Reply:
Yep. That's what I figured made most sense.

Reply:
I'm not entirely sure about this so lets just do it this way and then, if required, revisit it